src/main/java/FibonacciEven.java

```diff
@@ -0,0 +1,19 @@
+
+import java.util.List;
+
+public class FibonacciEven {
+
+    public static List<Integer> generateOnlyEvenFibonacciNumbersSum(int limitFromUser){
```

Comment:
Czemu zwracasz listę skoro potrzebny jest Ci tylko jeden numer - rezultat.

Reply:
Nazwa parametru nie jest poprawna - to co ta metoda przyjmuje to jest jakiś limit a nie numer od użytkownika.